docs/how-to/contribute-docs.rst

```diff
@@ -123,6 +130,37 @@ If a valid spelling of a word is identified as misspelled, then add the word to
 the list in ``docs/spelling_wordlist``. This will add the word to the
 spellchecker's dictionary.
 
+If you get an error related to SSL certificate verification::
+
+    Exception occurred:
+      File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/ssl.py", line 1342, in do_handshake
+        self._sslobj.do_handshake()
+    ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)
+
+The root certificate on your machine is out of date. You can correct this by
+installing the Python package `certifi`, and using that package to provide your
+SSL root certificate:
```

Comment:
This seems like a strange thing to happen...and to expect it to be common enough to document it here.

Whose cert is being trusted? Is `tox` masking some part of the environment potentially causing this?

Reply:
Ugh. No.

sphinxcontrib-spelling contains, amongst other gems, an XML-RPC call to a PyPI API that was deprecated 2 years ago. Python's `xmlrpc` module uses httplib, which uses a stale root trust certificate on macOS, because Apple uses libSSL rather than OpenSSL.

In short, computers were a mistake.